website/src/css/pages/product/_automation.scss

```diff
@@ -0,0 +1,30 @@
+.automation {
+   .container img {
+       display: inline-block;
+       width: 100% !important;
+    }
+
+    h1 {
+        display: inline-block;
+        position: relative;
+
+        &::after {
+            position: absolute;
+            top: -0.2rem;
+            right: -6.7rem;
+            content: 'Coming soon';
```

Comment:
Just to check, this is intended to look like superscript? 
![image](https://user-images.githubusercontent.com/5589410/63481235-afa51880-c449-11e9-9859-d6a6ea9c26c9.png)

Reply:
Let's say "private beta" instead of "coming soon".

Reply:
> Just to check, this is intended to look like superscript?

Yes. I'd love some design inspiration, it's just what I could come up with.